### PR TITLE
GitHub contribution help docs

### PR DESCRIPTION
**Pull request by christopher-hopper**
## Proposed Changes

- Add [Github Pull Request template](https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/) to help contributors describe their work
- Create a [Github Contributing guide](https://help.github.com/articles/setting-guidelines-for-repository-contributors/) shown as a link when people create a pull request
- Provde a [Github issue template](https://help.github.com/articles/creating-an-issue-template-for-your-repository/) to improve the quality of submitted issues

## Relates To

- Issue #255 

## Notify

- @alexdesignworks

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+If you haven't already, please take the time to read through the complete [Contributing guide][] for the project. It should help ensure your contributions succeed by aligning them with the goals for the project.
+
+**We want you to contribute the things you're excited about so everyone can benefit.**
+
+[Contributing guide]: http://beetbox.readthedocs.io/en/stable/contributing/contributing/
+
+Proposing Changes
+-----------------
+
+Please submit a [GitHub Pull Request to Beetbox][] with a clear list of what you've done.
```
Comment (AlexSkrypnyk) [inline .github/CONTRIBUTING.md]: broken link?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+- Fix
+
+## Relates To
+
+- Issue #0
+
+## Notify
+
+- @octocat
+- @beetboxvm/octoteam
```
Comment (AlexSkrypnyk) [inline .github/PULL_REQUEST_TEMPLATE.md]: is this a correct account?

**Review by AlexSkrypnyk — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (christopher-hopper) [reply .github/PULL_REQUEST_TEMPLATE.md]: As I'm not a member of the beetboxvm organisation, I can't see your / their teams. If I could, I'd tell you that is probably a non-existent team. 

Octocat is also an account you probably wouldn't notify. 

These are what are called examples.

Comment (AlexSkrypnyk) [reply .github/PULL_REQUEST_TEMPLATE.md]: cool, thanks for the explanation
@thom8 what would be the correct groups for notification

**Review by AlexSkrypnyk — APPROVED** [review]
Thanks for your contribution @christopher-hopper. Looks awesome!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (thom8) [reply .github/PULL_REQUEST_TEMPLATE.md]: @alexdesignworks we currently get notifications on the `drupalmel` slack channel so probably don't need this.

Anyone is welcome to join @ http://melbourne.drupal.org.au/

**Comment by christopher-hopper** [comment]
Is that documented anywhere?

**Comment by thom8** [comment]
It's more of an "unofficial" channel which most contributors are in, we have talked about creating an offical channel but haven't really felt the need so far.

**Comment by christopher-hopper** [comment]
Then let's come up with an official set of tools and standards that any new
contributor will understand. If someone sees parallel work and wants to
reach out to another GitHub contributor, they'll understand use of the @
notify to bring that person into the discussion

**Comment by christopher-hopper** [comment]
I'll remove the team @ mention as its obviously not needed here

**Comment by thom8** [comment]
sounds good -- slack has been a good back channel, it's sometimes a bit overkill to add an issue for a simple question.

Plus, it alerts us of all issues, PRs, commits, new releases (tags) & build status on Atlas.

**Review by thom8 — APPROVED** [review]

**Pull request merged**